src/inc/corjit.h

```diff
@@ -177,6 +177,8 @@ enum CorJitFlag2
     CORJIT_FLG2_SAMPLING_JIT_BACKGROUND = 0x00000001, // JIT is being invoked as a result of stack sampling for hot methods in the background
 #if COR_JIT_EE_VERSION > 460
     CORJIT_FLG2_USE_PINVOKE_HELPERS     = 0x00000002, // The JIT should use the PINVOKE_{BEGIN,END} helpers instead of emitting inline transitions
+    CORJIT_FLG2_ENABLE_INTRINSIC        = 0x00000004, // The JIT emits intrinsic as possible.
+    CORJIT_FLG2_DISABLE_INTRINSIC       = 0x00000008, // The JIT emits call instead of intrinsic.
```

Comment:
Under Minopts JIt always disables jit intrinsics for two reasons
1) intrinsics is considered an optimization.  If min opts is triggered due to a noway assert being hit, then we don't want to go through optimization paths because there is a chance we will hit the noway assert again.
2) Minopts is also turned on when generating debuggable code and intrinsics expansion throws debugger off.

What is the scenario under which VM needs to force jit intrinsics?